extensions/WEBGL_get_buffer_sub_data_async/extension.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
 
-<draft href="WEBGL_get_buffer_sub_data_async/">
+<rejected href="WEBGL_get_buffer_sub_data_async/">
```

Comment:
Has this file actually been moved to the rejected/ directory? That's needed in order to avoid breaking the Makefile-based build in extensions/ .